### PR TITLE
13.0 mig contract membership delegated partner

### PR DESCRIPTION
**Pull request by miguel-S73**
Standard migration 12.0 to 13.0

### CODE REVIEW

File: contract_membership_delegated_partner/__manifest__.py [hunk 1]
```diff
@@ -2,7 +2,7 @@
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 {
     "name": "Contract Membership Delegate Partner",
-    "version": "12.0.1.0.1",
+    "version": "13.0.1.0.1",
```
Comment (chienandalu) [inline contract_membership_delegated_partner/__manifest__.py]: It should be `13.0.1.0.0`

**Review by chienandalu — CHANGES_REQUESTED** [review]
Thnaks! Just a tiny detail:

File: contract_membership_delegated_partner/__manifest__.py (hunk 1 above)
Comment (miguel-S73) [reply contract_membership_delegated_partner/__manifest__.py]: fixed

**Review by chienandalu — APPROVED** [review]
Great :smile: :+1:

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**